src/types.ts

```diff
@@ -268,6 +299,12 @@ export interface ExcalidrawProps {
       nativeEvent: MouseEvent | React.PointerEvent<HTMLCanvasElement>;
     }>,
   ) => void;
+  renderCustomElementWidget?: (appState: AppState) => void;
+  customElementsConfig?: CustomElementConfig[];
```

Comment:
let's make this into ` customElementsConfig?: Record<CustomType, CustomElementConfig>` so that we don't have to look up specific configs in an array.